src/assign.js

```diff
@@ -10,5 +11,5 @@ module.exports = function assign (/* objects list */) {
     }
 
     return memo
-  }, {})
+  }, initial || {})
```

Comment:
I needed to be able to merge properties into an object, which resulted in a updates to how `assign` was used in a couple of places. I may actually replace this with the `object-assign` NPM package. I'm evaluating browser support for some of the Object methods it uses.